src/config.rs

```diff
@@ -93,6 +93,10 @@ pub struct DbOptions {
     /// allows such readers to detect new L0+ files.**
     pub manifest_poll_interval: Duration,
 
+    /// Maximum number of tasks that can be run in parallel to limit resource consumption
+    /// This is used when fetching WALs in parallel on recovery.
+    pub maximum_parallel_tasks_limit: usize,
```

Comment:
TBH, I'd rather just hard code the parallel tasks. I want to be mindful that we don't have too many knobs to tune with Slate. IMO, this is one of the drawbacks of RocksDB.